browser/brave_shields/brave_timestamp_rounding_browsertest.cc

```diff
@@ -0,0 +1,111 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <math.h>
+#include <string>
+
+// Prevent a -Wimplicit-const-int-float-conversion warning:
+#define fmod(A, B) fmod(A, static_cast<double>(kMaxULL) + 1.0)
+#include "third_party/blink/renderer/bindings/core/v8/v8_binding_for_core.h"
```

Comment:
gncheck is right: `Can't include this header from here.`
this file is internal to blink, it should not appear here.

Reply:
I wonder when this error appears? `Prevent a -Wimplicit-const-int-float-conversion warning:`

Reply:
Removed this -- will test in qa-test-pages instead

Reply:
Removed